src/vorta/network_status/darwin.py

```diff
@@ -1,53 +1,82 @@
 import subprocess
 from datetime import datetime as dt
-from typing import Iterator, Optional
+from typing import Iterator, List, Optional
+
+from CoreWLAN import CWInterface, CWNetwork, CWWiFiClient
```

Comment:
I was wondering whether this toplevel import is a good practice since the dependency is optional.

Reply:
Are you suggestion suppressing import errors like what's happening in the `notifications` module?

https://github.com/borgbase/vorta/blob/master/src/vorta/notifications.py#L8-L11

I have a Linux host I can test with.  I'll see if I can do that soon to see if the application throws an ImportError for me.

Reply:
I can report that on Pop_OS 22.04 I was able to start the app in development mode and do a successful backup to a remote host.  Also the full test matrix passed for me via `make test-unit` on the same host.

I saw an INFO log message: `Using NetworkManagerMonitor NetworkStatusMonitor implementation`

No INFO or DEBUG logs mentioning the new imports.

I do see some DEBUG log messages from `vorta.keyring.abc` about `No module named 'objc'`, but they have not affected the manual backup test.

I'm happy to make the change if we want it.  But wanted to report back these manual testing results.

Reply:
An upshot of my local testing was that I also tried a backup while tethered to my iOS device and it detected the metered connection. 🎉